doc/source/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.rst

```diff
@@ -262,12 +253,13 @@ where :math:`U_\infty` is the upstream velocity and :math:`D` is the diameter of
 
 .. code-block:: text
 
-  cells     C_D       
-   1167    13.20  
-   2247    13.93  
-   4302    14.15  
-   8268    14.23  
-  15990    14.24  
+ Cells   C_D
+ 1512    13.26
+ 2874    13.94
+ 5460    14.16
+ 10374   14.18
+ 19521   14.22
+
```

Comment:
I did not get those results after running the example and the python script, see below : 
![image](https://github.com/user-attachments/assets/c7ef4f3e-8b11-48b8-93cf-87eaf0aa742d)

Reply:
Did you remake the mesh with gmsh because I get the following results:
Cells   Cd
1512.0   13.2626322188
2874.0   13.945769007
5460.0   14.1672078172
10374.0   14.1869390276
19521.0   14.2221387622